index.html

```diff
@@ -4237,6 +4237,38 @@ <h2 id="examples-tzigla">Tzigla</h2>
 
     <h2 id="changelog">Change Log</h2>
 
+    <b class="header">1.2.1</b> &mdash; <small><i>Jun. 3, 2015</i></small>
+    &mdash; <a href="https://github.com/jashkenas/backbone/compare/1.2.0...1.2.1">Diff</a>
+    &mdash; <a href="https://cdn.rawgit.com/jashkenas/backbone/1.2.1/index.html">Docs</a>
+    <br />
+    <ul style="margin-top: 5px;">
+      <li>
+        <tt>Collection#add</tt> will now avoid trying to parse a model instance with <tt>parse : true</tt>.
+      </li>
+      <li>
+        Bug fix where <tt>Collection#remove</tt> was not actuately returning only the models
+        removed from the collection.
+      </li>
+      <li>
+        <tt>Model#save</tt> will no longer mutate the passed in attribute hash with the server's
+        response when <tt>wait: true</tt>.
+      </li>
+      <li>
+        Any passed <tt>options.attrs</tt> are now respected when fetching a model with <tt>patch: true</tt>.
```

Comment:
Ah yeah.. my bad there.  I had thought I was copy-modifying the log note from the related bugfix that went out with `1.2.0`, but now it looks like that may not have been listed in the changelog at all.